core.lua

```diff
@@ -4267,22 +4265,23 @@ do
                     if isExtendedProfile then
                         if showRaidEncounters then
                             local raidProvider = provider:GetProviderByType(ns.PROVIDER_DATA_TYPE.Raid, state.region)
-                            for i = 1, raidProvider.currentRaid.bossCount do
+                            local currentRaid = raidProvider.currentRaids[1] -- TODO: figure out which raid to look at
```

Comment:
@Vladinator - Here is a specific spot we'll need to handle. I made it just pull the first raid for now.